pytometry/plotting/_scatter_density.py

```diff
@@ -54,6 +53,8 @@ def scatter_density(
             draw into an existing figure.
         figsize (tuple), optional:
             Figure size (width, height) if ``ax`` not provided. Defaults to (10, 10).
+        bins (int or tuple), optional:
```

Comment:
This is a general point, but I don't see the point of typing docstrings when you're using typehints in the function stub (as you should!). It's annoying to keep both in sync. Sphinx picks up the types from the function header anyways.

Reply:
I totally agree, I'll open a separate issue!